### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by iamalhera**
Homepage-Cali Shop

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        </div>
+
+        <!-- Hero Section -->
+        <div class="hero-section">
+            <div class="hero-img hero-1 flex-center">
+                <img class="" src="/src/assets/hero-image.jpg" alt="Hero Section Image"></img>
+                <div class="hero-overlay flex-column">
+                    <h1>WORK ETHICS OF ATHELETE</h1>
+                    <div class="text-md-size margin-tb-sm">find the right attitude and products of top fitness coach
+                    </div>
```
Comment (Ankur9669) [inline index.html:61]: Can use p tag here

File: index.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+                    <button class="btn btn-primary">Shop Accessories</button>
+                </div>
+            </div>
+        </div>
+
+        <!-- Featured Category -->
+        <div class="featured-category flex-column">
+            <div class="flex-column padder-md">
+                <p class="h1">Featured Category</p>
+                <p class="text-rg-size">Check out our favourite categories for your fitness</p>
```
Comment (Ankur9669) [inline index.html:71]: Featured category is a heading so you can use h tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+                    <p class="text-lg-size text-bold">TRAINING EQUIPMENT</p>
+                </div>
+            </div>
+        </div>
+
+        <!-- New Release -->
+        <div class="featured-category flex-column">
+            <div class="flex-column padder-md">
+                <p class="h1">New Release</p>
+                <p class="text-rg-size">Check out our new release, The quickest way to top shape</p>
```
Comment (Ankur9669) [inline index.html:89]: Same here

File: src/pages/homepage.css [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+    color: var(--ca-primary-color); 
+}
+.btn{
+    padding: 0.7rem 1rem;
+    border-radius: 5px;
+    font-size: 1rem;
+    font-weight: 200;
+    color : var(--ca-primary-color);
+    background-color: white;
+    border : none ;
```
Comment (Ankur9669) [inline src/pages/homepage.css:58]: If possible use css variable

File: src/pages/homepage.css [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+    right : -1rem ;
+    color : white ;
+
+}
+
+
+
+
+/* hero-section  */
+.hero-section{
```
Comment (FarhanMobashir) [inline src/pages/homepage.css:105]: remove redundant code

File: src/pages/homepage.css [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+    padding : 0.5rem 1rem ;
+}
+.search-bar{
+    background-color: var(--ca-hover-color);
+    padding : 0.3rem ;
+    border: none;
+}
+.header-right-side{
+    gap : rem ;
+    margin-right : 0.5rem ;
```
Comment (Ankur9669) [inline src/pages/homepage.css:81]: only rem??

File: src/pages/homepage.css [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+    background-color : var(--ca-icon-content-bg-color);
+    padding : 0.8rem ;
+    height : 0.5rem ;
+    width : 0.5rem ;
+    font-size : 0.8rem ;
+    border-radius : 50% ;
+    top : -1rem ;
+    right : -1rem ;
+    color : white ;
+
```
Comment (Ankur9669) [inline src/pages/homepage.css:98]: Can use css variable

**Review by FarhanMobashir — COMMENTED** [review]
code is neat just make the suggested changes

File: src/pages/homepage.css [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+}
+
+
+
+
+/* hero-section  */
+.hero-section{
+
+}
+.hero-img img{
```
Comment (Ankur9669) [inline src/pages/homepage.css:108]: Can remove empty classes

File: src/pages/homepage.css [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+}
+.btn-primary{
+    background-color: var(--ca-primary-color);
+    color : white ;
+}
+.btn-primary:hover{
+    background-color: var(--ca-primary-color-lighter);
+    color : white ;
+}
+
```
Comment (Ankur9669) [inline src/pages/homepage.css:127]: Use css variables here

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
The UI looks great and the code is also fine. But i have a question have you used your component librarary here?

File: src/pages/homepage.css (hunk 5 above)
Comment (iamalhera) [reply src/pages/homepage.css:105]: done

File: index.html (hunk 1 above)
Comment (iamalhera) [reply index.html:61]: ok

File: index.html (hunk 2 above)
Comment (iamalhera) [reply index.html:71]: ohh noticed just now

File: src/pages/homepage.css (hunk 6 above)
Comment (iamalhera) [reply src/pages/homepage.css:81]: ohh! missed while editing i think

**Comment by iamalhera** [comment]
> The UI looks great and the code is also fine. But i have a question have you used your component librarary here?

Ok so i started using codes of component library, the UI which i was building starts getting error, so i started taking out components of my library and started making a generalized code and the UI, So as soon as i will complete the E-Com UI, i will rectify and change all brute codes

File: index.html [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+                </div>
+                <div class="flex-column category-item">
+                    <img class="category-item-img" src="/src/assets/new-release-3.png" alt="weighted vest"></img>
+                    <p class="text-lg-size text-bold">WEIGHTED VEST (10KG WT)</p>
+                </div>
+            </div>
+        </div>
+
+        <!-- Footer -->
+        <footer class="page-footer dark-bg flex-column padder-tb-lg">
```
Comment (AshutoshM29) [inline index.html:108]: Just to keep your code uniform, you should use class name as footer-page instead of page-footer

**Review by AshutoshM29 — CHANGES_REQUESTED** [review]
Rest looks good to merge

**Pull request merged**